apiserver/facades/client/charms/client.go

```diff
@@ -265,7 +265,7 @@ func (a *API) addCharmWithAuthorization(args params.AddCharmWithAuth) (params.Ch
 	}
 
 	if args.Origin.Base.Name == "" || args.Origin.Base.Channel == "" {
-		return params.CharmOriginResult{}, errors.BadRequestf("base required for charm-hub charms")
+		return params.CharmOriginResult{}, errors.BadRequestf("base required for Charmhub charms")
```

Comment:
We should probably abstract out (i.e. create and use constants for) all the uses of proper nouns in the repository so it can be easily changed based on any changes in any branding policy. Should probably be done in another pR.

Reply:
Dora will be working on making Charmhub consistent across juju output. Adding a constant is a mixed bag, nice in this case, but maybe hard to get people to use ALL the time.